slice.py

```diff
@@ -14,55 +63,50 @@ def linearInterpolation(p1, p2, distance):
     :param distance: Between 0 and 1, Lower numbers return points closer to p1.
     :return: A point on the line between p1 and p2
     '''
-    slopex = (p1[0] - p2[0])
```

Comment:
I think the body of this function can be replaced with `return p1 * (1-distance) + p2 * distance`

Reply:
https://github.com/youngkiu/stl-to-voxel/blob/mesh_numpy/slice.py#L71
I am preparing a new PR, can I review it there?